go.mod

```diff
@@ -1,6 +1,6 @@
 module go.temporal.io/server
 
-go 1.17
+go 1.18
```

Comment:
This doesn't have to be changed in order to compile temporal with 1.18.

Is there anything in 1.18 that would require this change?

Reply:
I am pretty sure we will use many 1.18 features very soon.